backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/starrating/BpkStarRating.kt

```diff
@@ -61,6 +62,7 @@ fun BpkStarRating(
     modifier: Modifier = Modifier,
     rounding: BpkRatingRounding = BpkRatingRounding.Down,
     size: BpkStarRatingSize = BpkStarRatingSize.Small,
+    color: Color? = null,
```

Comment:
where's this coming from? looking at figma the colour changed in general for the hotel star, rather than it being configurable.

Reply:
The designers have discussed here before. https://skyscanner.slack.com/archives/CUQ587E5D/p1732083779847409

Reply:
yeah i think web is different as they don't have a different variant for hotel star rating, but we do on app (mainly for accessibility to ensure it still says 3/5 for example)